test/e2e/storage/csi_volumes.go

```diff
@@ -45,6 +48,7 @@ import (
 // List of testDrivers to be executed in below loop
 var csiTestDrivers = []func() drivers.TestDriver{
 	drivers.InitHostPathCSIDriver,
+	drivers.InitMockCSIDriver,
```

Comment:
@bertinatto 
should we remove this? How is the mock driver passing
testsuites.InitVolumesTestSuite,
testsuites.InitVolumeIOTestSuite,
testsuites.InitVolumeModeTestSuite,
testsuites.InitSubPathTestSuite,
testsuites.InitProvisioningTestSuite, (currently CI is failing on this one)
?

I'm not sure how the non-provisioning tests are passing/being skipped, there's lots of layers of abstraction to wade through now (too many imo :])

Reply:
I removed mock driver from the other tests, it does not provide persistent storage that the tests expect. It's now used only in passing workload info to the driver.